src/example.jl

```diff
@@ -1,32 +1,24 @@
-""" The Academic Performance Index is computed for all California schools based on standardised testing of students. The data sets contain information for all schools with at least 100 students and for various probability samples of the data.
+const PKG_DIR = joinpath(pathof(Survey), "..", "..") |> normpath
+asset_path(file) = joinpath(PKG_DIR, "assets", file)
```

Comment:
Should `asset_path` function moved inside `load_data`? This was `Survey.asset_path` is not available to other functions. 
@chiraganand @greimel 

Sorry for being late on this.

Reply:
I don't think that this would be idiomatic in Julia. Most packages have tons of internal functions. As long as it isn't documented, users should not rely on it.

Reply:
Yes, as long as you don't export the function or document it nobody should be using it.